Snowflake.Data/Client/SnowflakeDbConnection.cs

```diff
@@ -100,9 +141,36 @@ public override void Close()
             _connectionState = ConnectionState.Closed;
         }
 
+        public override void Close()

```

Comment:
is Close() multi thread safe?

Reply:
DBConnection subclasses aren't expected to be threadsafe.